### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by jazbassi1**
added solution for iterables and iterations hackerrank

### CODE REVIEW

File: IterablesandIterators.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from itertools import combinations
```
Comment (manokhina) [inline IterablesandIterators.py:47]: Could you provide the context for the problem? i.e. what is done and why

File: IterablesandIterators.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from itertools import combinations
+
+n = int(input())
+l = list(input().split())
+k = int(input())
+c = list(combinations(l, k)) # combination of lists
+r = [i for i in c if 'a' in i]
+print('%.3f'%(len(r)/ len(c)))
```
Comment (manokhina) [inline IterablesandIterators.py]: empty line needed

**Review by manokhina — CHANGES_REQUESTED** [review]
Please add context for the problem solved

File: IterablesandIterators.py (hunk 1 above)
Comment (jazbassi1) [reply IterablesandIterators.py:47]: Changes made

**Pull request merged**